features/in_onderzoek.feature

```diff
@@ -7,134 +7,738 @@ Functionaliteit: in onderzoek
+      | datumIngangOnderzoek          | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                                                                              |
+      | type                | RaadpleegMetBurgerservicenummer                                                                                     |
+      | burgerservicenummer | 555550001                                                                                                           |
+      | fields              | naam.voornamen,naam.adellijkeTitelPredicaat,naam.voorvoegsel,naam.geslachtsnaam,naam.aanduidingNaamgebruik,geboorte |
+      Dan heeft de persoon met burgerservicenummer '555550001' alleen de volgende 'naam.inOnderzoek' gegevens
+      | naam                    | waarde     |
+      | voornamen               | true       |
+      | adellijkeTitelPredicaat | true       |
+      | voorvoegsel             | true       |
+      | geslachtsnaam           | true       |
+      | datumIngangOnderzoek    | 2022-03-07 |
+      En heeft de persoon GEEN 'inOnderzoek' gegevens
+      En heeft de persoon GEEN 'geboorte.inOnderzoek' gegevens
+
+    Scenario: kind heeft hele groep geboorte in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 555550001 |
+      En de persoon heeft een kind met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550002 |
+      En het kind met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
+      | naam                          | waarde   |
+      | aanduidingGegevensInOnderzoek | 090300   |
+      | datumIngangOnderzoek          | 20220307 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | kinderen                        |
+      Dan heeft het kind met burgerservicenummer '555550002' alleen de volgende 'geboorte.inOnderzoek' gegevens
+      | naam                 | waarde     |
+      | datum                | true       |
+      | plaats               | true       |
+      | land                 | true       |
+      | datumIngangOnderzoek | 2022-03-07 |
+      En heeft het kind met burgerservicenummer '555550002' GEEN 'inOnderzoek' gegevens
+      En heeft het kind met burgerservicenummer '555550002' GEEN 'naam.inOnderzoek' gegevens
+      En heeft de persoon met burgerservicenummer '555550001' GEEN 'inOnderzoek' gegevens
+      En heeft de persoon met burgerservicenummer '555550001' GEEN 'geboorte.inOnderzoek' gegevens
+
+    Scenario: partner heeft hele groep soort verbintenis in onderzoek
```

Comment:
Het zal wellicht liggen aan de wijze waarop een PL is opgesteld maar ik begrijp niet dat je bij 'soort verbintenis' spreekt over een groep. Ik heb nl. het idee dat het gewoon om een veld gaat. Geeft het volgende scenario (1e regel gewijzigd en ' aanduidingGegevensInOnderzoek' heeft hier de waarde '051510') niet hetzelfde resultaat?

```
    Scenario: partner heeft het veld soort verbintenis in onderzoek
      Gegeven het systeem heeft een persoon met de volgende gegevens
      | naam                        | waarde    |
      | burgerservicenummer         | 555550001 |
      En de persoon heeft een partner met de volgende gegevens
      | naam                | waarde    |
      | burgerservicenummer | 555550002 |
      En de partner met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
      | naam                          | waarde   |
      | aanduidingGegevensInOnderzoek | 051510   |
      | datumIngangOnderzoek          | 20220307 |
      Als personen wordt gezocht met de volgende parameters
      | naam                | waarde                          |
      | type                | RaadpleegMetBurgerservicenummer |
      | burgerservicenummer | 555550001                       |
      | fields              | partners                        |
      Dan heeft de partner met burgerservicenummer '555550002' alleen de volgende 'inOnderzoek' gegevens
      | naam                 | waarde     |
      | soortVerbintenis     | true       |
      | datumIngangOnderzoek | 2022-03-07 |
      En heeft de partner met burgerservicenummer '555550002' GEEN 'naam.inOnderzoek' gegevens
      En heeft de partner met burgerservicenummer '555550002' GEEN 'geboorte.inOnderzoek' gegevens
      En heeft de persoon met burgerservicenummer '555550001' GEEN 'inOnderzoek' gegevens
```

Reply:
tja, de wondere wereld van LO GBA. Bekijk die Excel sheet nog maar eens, dan zie je dat dit vaker voorkomt. Soms heb je een groep waar maar 1 element in zit. Dan kan dat gegeven in onderzoek gezet zijn door het specifieke element in onderzoek te zetten, of door die groep in onderzoek te zetten. Dus aanduiding in onderzoek 051500 en 051510 hebben hetzelfde effect. De feature moet het zo beschrijven dat allebei de vormen goed werken.

Ik maak er een abstract scenario van die beide variante laat zien